extensions/wikia/AdEngine/js/lookup/prebid/prebidSettings.js

```diff
@@ -0,0 +1,60 @@
+/*global define*/
+define('ext.wikia.adEngine.lookup.prebid.prebidSettings', [], function () {
+	'use strict';
+
+	/*
+	 * Documentation:
+	 * - http://prebid.org/dev-docs/publisher-api-reference.html#module_pbjs.setPriceGranularity
+	 * - http://prebid.org/dev-docs/publisher-api-reference.html#module_pbjs.bidderSettings
+	 * - http://prebid.org/dev-docs/examples/custom-price-bucket.html
+	 */
+	function transformPrice(bidResponse) {
+		var cpm = bidResponse.cpm,
```

Comment:
is it possible CPM is not set? Maybe we should proof our solution for such case.